autoware_launch/launch/components/tier4_perception_component.launch.xml

```diff
@@ -19,12 +19,13 @@
   <arg name="detected_objects_filter_method" default="lanelet_filter" description="options: lanelet_filter, position_filter"/>
   <arg name="detected_objects_validation_method" default="obstacle_pointcloud" description="options: obstacle_pointcloud, occupancy_grid (must be used with laserscan_based_occupancy_grid_map)"/>
   <arg name="data_path" default="$(env HOME)/autoware_data" description="packages data and artifacts directory path"/>
-  <arg name="lidar_detection_model" default="centerpoint" description="options: `transfusion`, `centerpoint`, `apollo`, `pointpainting`, `clustering`"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="specify lidar detection model type and optionally can add model name like `centerpoint/centerpoint_sigma`"/>
```

Comment:
I prefer to add following statement.
"If the model name is not declared, the default model in perception.launch.xml will be used. "